Add cis benchmarks (#118)
giantswarm/starboard-exporter: README.md

## Metrics

This exporter exposes two types of metrics:
This exporter exposes various types of metrics:

> ### Summary

Reviewer: I'd suggest we add some hierarchy now, since we also have ConfigAuditReports not shown here.
What about something like:
```
## Metrics
This exporter exposes several types of metrics:

### CIS Benchmarks
#### Report Summary
Exposes an overview of the entire report...
#### Section Summary
For slightly more granular visibility...
#### Result Details
Exposes detailed information about the results of each test...

### Vulnerability Reports
#### Report Summary
...
#### Vulnerability Details
```
One of us can add configauditreports later

Author: Done 🎸

## Metrics

This exporter exposes several types of metrics:

### CIS Benchmarks

#### Report Summary

> A summary series exposes the count of checks of each status reported in a given `CISKubeBenchReport`. For example:

Reviewer: ```suggestion
A report summary series exposes the count of checks of each status reported in a given `CISKubeBenchReport`. For example:
```

```shell
starboard_exporter_ciskubebenchreport_report_summary_count{
    node_name="bj56o-master-bj56o-000000"
    status="FAIL"
    } 31
```

#### Section Summary

> A summary series exposes the count of checks of each status reported in a given `CISKubeBenchSection`. For example:

Reviewer: ```suggestion
For slightly more granular reporting, a section summary series exposes the count of checks of each status reported in a given `CISKubeBenchSection`. For example:
```

```shell
starboard_exporter_ciskubebenchreport_section_summary_count{
    node_name="bj56o-master-bj56o-000000"
    node_type="controlplane"
    section_name="Control Plane Configuration"
    status="WARN"
    } 4
```

#### Result Detail

> A detail for CIS benchmark results series exposes field from each instance of an Aqua `CISKubeBenchResult`. For example:

Reviewer: ```suggestion
A CIS benchmark result info series exposes fields from each instance of an Aqua `CISKubeBenchResult`. For example:
```

### Detail / Vulnerability

> A detail for vulnerability series exposes fields from each instance of an Aqua `Vulnerability`. The value of the metric is the `Score` for the vulnerability. For example:

Reviewer: This was correct but you're right was not totally clear
```suggestion
A "detail" or "vulnerability" series exposes fields from each instance of an Aqua `Vulnerability`. The value of the metric is the `Score` for the vulnerability. For example:
```

      - ...
```

> To enable an additional detail series *per CIS Benchmark Result*, use the `--cis-detail-report-labels` flag to specify which labels should be exposed. For example:

Reviewer: ```suggestion
The same can be done for CIS Benchmark Results. To enable an additional detail series *per CIS Benchmark Result*, use the `--cis-detail-report-labels` flag to specify which labels should be exposed. For example:
```
